crawl-ref/source/spl-damage.cc

```diff
@@ -2699,7 +2699,7 @@ void toxic_radiance_effect(actor* agent, int mult, bool on_cast)
         if (agent->is_monster() && mons_aligned(agent, *ai))
             continue;
 
-        int dam = roll_dice(1, 1 + pow / 20) * mult;
+        int dam = roll_dice(1, 1 + pow / 20) * mult / 10;
```

Comment:
This line isn't right, for delays less than 10 aut it will cause OTR to do zero damage. It should be

```suggestion
        int dam = roll_dice(1, 1 + pow / 20) * div_rand_round(mult, 10);
```

in order to match the old behavior in expected value